README.md

```diff
@@ -1,7 +1,49 @@
-# Iroha JavaScript
+# Iroha2 JavaScript
 
 JavaScript SDK for [Iroha 2](https://github.com/hyperledger/iroha/tree/iroha2/).
 
+The Iroha 2 JavaScript library includes the following packages:
+
+- [`@iroha2/client`](./packages/client/) submits requests to the Iroha peer
+- [`@iroha2/monorepo-crypto`](./packages/crypto/):
+  - [`@iroha2/crypto-core`](./packages/crypto/core/) provides unified crypto interface for Iroha 2
+  - [`@iroha2/crypto-target-node`](./packages/crypto/target-node/) provides compiled crypto WASM for the Node.js environment
+  - [`@iroha2/crypto-target-web`](./packages/crypto/target-web/) provides compiled crypto WASM for native Web (ESM)
+  - [`@iroha2/crypto-target-bundler`](./packages/crypto/target-bundler/) provides compiled crypto WASM to use with bundlers such as Webpack
+- [`@iroha2/data-model`](./packages/data-model/) provides SCALE codecs for Iroha 2 data model
+  
+Other packages you can find in this repository:
+- [`@iroha2/client-isomorphic-fetch`](./packages/client-isomorphic-fetch/) fallbacks to `node-fetch` or native `fetch()` in Node.js
+- [`@iroha2/client-isomorphic-ws`](./packages/client-isomorphic-ws/) contains isomorphic WebSocket transport for `@iroha2/client`
+- [`@iroha2/docs-recipes`](./packages/docs-recipes/) contains code samples used in documentation
+- [`@iroha2/data-model-schema`](./packages/data-model-schema/) contains Iroha 2 Schema
+- [`@iroha2/i64-fixnum`](./packages/i64-fixnum/) handles operations with fixed-point numbers
+- [`@iroha2/test-peer`](./packages/test-peer/) provides a wrapper around the `iroha_cli` with an easy-to-use interface for working with the Iroha peer
```

Comment:
`@iroha2/test-peer` shouldn't be documented here, I think. It is an internal service package for testing purposes. It is a private package, and cannot be used outside of the repo.

Reply:
```suggestion
```